presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveStorageFormats.java

```diff
@@ -0,0 +1,8 @@
+package com.facebook.presto.tests.hive;
+
+/**
+ * Created by andrii on 21.03.16.
```

Comment:
It is auto-generated by my IDE, and i just forgot to remove it. This is lame, i know. Will remove.